packages/core-components/src/lib/atoms/Dropdown.svelte

```diff
@@ -32,7 +32,10 @@
 	$: component_identifier = `_Dropdown-${name}`;
 
 	const search = (query) =>
-		profile(db.query, query, component_identifier, (value) => (data = value));
+		profile(db.query, query, {
+			query_name: component_identifier,
+			callback: (value) => (data = value)
+		});
```

Comment:
Does passing an object here work? I've always written this as
```suggestion
		profile(db.query, query, component_identifier, (value) => (data = value) );
```

Reply:
Yeah, I just switched it to accept an object instead of an arbitrary amount of parameters so it's easier to add onto in the future

Reply:
An array would be a better choice than an object in this case, ordering is important; I think this would be an okay one to refactor in the future if we need to though